docs/resources/Channel.md

```diff
@@ -77,6 +78,13 @@ Represents a guild or DM channel within Discord.
 | PINNED      | 1 << 1 | this thread is pinned to the top of its parent `GUILD_FORUM` channel                                                                         |
 | REQUIRE_TAG | 1 << 4 | whether a tag is required to be specified when creating a thread in a `GUILD_FORUM` channel. Tags are specified in the `applied_tags` field. |
 
+###### Sort Order Modes
```

Comment:
```suggestion
###### Sort Order Types
```

Reply:
unless there was specific reason for "Modes"

Reply:
I just took the naming from the table above called "Video Quality Modes"

Reply:
mmmm i think i disagree with that one too. most of those constant tables use "Types"—Allowed Mentions, Channels, Message, etc.